### PR TITLE
Lazy install Paddle wheels

### PR DESCRIPTION
**Pull request by reyoung**
* install wheels when invoke paddle script if current python don't
  have paddle packages, or installed a previous version.

### CODE REVIEW

File: paddle/scripts/submit_local.sh.in [hunk 1]
```diff
@@ -43,6 +43,38 @@ fi
...
+else:
+  sys.exit(0)
+EOF
+
+if [ $? -eq 1 ]; then  # Older version installed, or not installed at all
+   set -e
+   echo "First time run paddle, need to install some python dependencies."
+   BASEDIR=$(dirname "$0")
+   set -x
+   pip install ${BASEDIR}/../opt/paddle/share/wheels/*.whl
```
Comment (emailweixu) [inline paddle/scripts/submit_local.sh.in]: The use may not have the permission to do pip install. Need to check whether it's successfully installed.

File: paddle/scripts/submit_local.sh.in [hunk 2]
```diff
@@ -43,6 +43,40 @@ fi
...
+EOF
+
+if [ $? -eq 1 ]; then  # Older version installed, or not installed at all
+   echo "First time run paddle, need to install some python dependencies."
+   BASEDIR=$(dirname "$0")
+   pip install ${BASEDIR}/../opt/paddle/share/wheels/*.whl
+   if [ $? -ne 0 ]; then
+      echo "pip install wheels failed. "
+      echo "Please use 'sudo paddle' at the first time you use PaddlePaddle"
+      echo "PaddlePaddle will install some python dependencies automatically."
```
Comment (reyoung) [inline paddle/scripts/submit_local.sh.in]: Add an error logs here.

File: .travis.yml [hunk 3]
```diff
@@ -29,6 +29,7 @@ before_install:
 script:
   - paddle/scripts/travis/build.sh
   - paddle/scripts/travis/unittest.sh
+  - paddle/scripts/travis/make_install.sh
```
Comment (reyoung) [inline .travis.yml]: also test make install during travis-ci

File: paddle/scripts/travis/make_install.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+cd `dirname $0`
+source ./common.sh
+sudo make install
```
Comment (reyoung) [inline paddle/scripts/travis/make_install.sh]: test make install

File: paddle/scripts/submit_local.sh.in (hunk 2 above)
Comment (emailweixu) [inline paddle/scripts/submit_local.sh.in]: No point to run the rest, should exit here.

File: paddle/scripts/submit_local.sh.in [hunk 5]
```diff
@@ -43,6 +43,41 @@ fi
...
+      exit 1
```
Comment (reyoung) [inline paddle/scripts/submit_local.sh.in]: Done.

**Pull request merged**